eZ/Publish/Core/Persistence/Legacy/Tests/Content/FieldValue/Converter/DateAndTimeTest.php

```diff
@@ -256,10 +256,14 @@ public function testToFieldDefinitionCurrentDate()
         );
 
         $this->converter->toFieldDefinition($storageDef, $fieldDef);
+        sleep(2);
```

Comment:
@andrerom Cool, with that I have gotten it to work. It's clumsy though. It's not enough to override `time()`. The `DateTime` constructor isn't based on that, so I have to rewrite not just the test but also the tested code, to work around this. More trouble than it's worth?

Alternative: The time-dependent bit is very small, so I think the test will be stable even with just 1 second of sleep. Would that be ok?